src/Symfony/Upgrade/Fixer/YamlFixer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Symfony\Upgrade\Fixer;
+
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Yaml;
+
+class YamlFixer extends AbstractFixer
+{
+    public function fix(\SplFileInfo $file, $content)
+    {
+        if ('yml' !== strtolower($file->getExtension())) {
+            return $content;
+        }
+
+        try {
+            $array = Yaml::parse($content);
+        } catch (ParseException $ex) {
+
+            // Skipping invalid YAML files
+            return $content;
+        }
+
+        $fixed = Yaml::dump($array);
+
+        return $fixed;
+    }
+
+    public function getDescription()
+    {
+        return 'This fixer strictly escapes YAML files.';
```

Comment:
Try something more consistent to what we have in [README](https://github.com/umpirsky/Symfony-Upgrade-Fixer#fixers-available), because we generate readme file from this descriptions. ;)